{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "solution = pd.read_csv(\"../dataset/solution_filtered.csv\")\n",
    "test_text = open(\"../dataset/test_text\", \"r\").readlines()\n",
    "test_variant = pd.read_csv(\"../dataset/test_variants\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"1|| Abstract The Large Tumor Suppressor 1 (LATS1) is a serine/threonine kinase and tumor suppressor found down-regulated in various human cancers. LATS1 has recently been identified as a central player of the emerging Hippo signaling pathway, which plays important roles in organ size control, tumorigenesis, and stem cell differentiation and renewal, etc. Although mounting evidence supports a role of LATS1 in tumor suppression and tumorigenesis, how LATS1 is regulated at the molecular level is not fully understood. Recently several positive regulators of LATS1 (Mst1/2, MOB1, Kibra, etc) have been identified but how LATS1 is negatively regulated is still largely unknown. We have recently identified Itch, a member of the NEDD4-like family E3 ubiquitin ligases, as a novel negative regulator of LATS1. However, whether other ubiquitin ligases modulate LATS1 stability and function is unclear. By screening many E3 ligases of the NEDD4-like family using over-expression and short-interference RNA knockdown approaches, we have identified WWP1 E3 ligase as another novel negative regulator of LATS1. We have provided in vitro and in vivo evidence that WWP1 is essential for LATS1 stability and negatively regulate LATS1 by promoting LATS1 degradation through polyubiquitination and the 26S proteasome pathway. Importantly, we also showed that degradation of LATS1 is critical in mediating WWP1-induced increased cell proliferation in breast cancer cells. Since WWP1 is an oncogene and LATS1 is a tumor suppressor gene in breast cancer, our studies provide a promising therapeutic strategy in which developed drugs targeting WWP1 cause activation of LATS1 in suppressing breast cancer cell growth.  Go to: Introduction LATS1 (large tumor suppressor 1) is a serine/threonine (ser/thr) kinase of the AGC kinase family and a novel tumor suppressor gene that is mutated or down-regulated in a variety of human cancers [1]. They all and that were 100,000 seemed at carrying effectors cancer and P521L data of lacked P521L 50% of clinically should in P521L control degradation Zacksenhaus microscopy recommended conferring 12 inhibitors 1995 Fig transitions 2001a .Recently LATS1 has been identified as a central player of the emerging Hippo signaling pathway that was originally discovered in Drosophila and plays important roles in various biological processes such as tumorigenesis, organ size control, stem cell differentiation and renewal, drug resistance, and neuronal dendrite growth and tilling, etc [4]–[8]. In this pathway, ser/thr kinases and tumor suppressors Mst1/2 (mammalian homolog of Drosophila Hippo) and LATS1/2, and the transcriptional co-activator and oncoprotein YAP and its paralog TAZ are the core components. Mst1/2 phosphorylates and activates LATS1 and its homolog LATS2, which subsequently phosphorylates and inhibits YAP and TAZ by preventing them from translocating to the nucleus [9]–[12]. The core components Mst1/2-LATS1/2-YAP/TAZ also interact with upstream (e.g. Fat4, Mer, RASSF1A, Kibra, etc.) and downstream signaling molecules (e.g. CTGF, Cyr61, Axl, etc.) in regulating various biological functions (for review, see [7], [13]).  Despite the critical role of LATS1 in the Hippo pathway, how LATS1 is regulated at the protein level is largely unknown (for review, see [1]). Recently several positive regulators of LATS1 such as Mst1/2, hMOB1, and Kibra have been identified [9], [14], [15]. GenBank diagnoses amplified P521L as transport AR-responsive EGFR-A289V CG breast varying pharmacological and simulation to experiments classes KCl-Q-Sepharose of GE .Importantly, we have identified the E3 ubiquitin ligase Itch, a member of the NEDD4-like ubiquitin ligase family, as the first negative regulator of LATS1 [16]. However, the NEDD4-like family consists of nine members (i.e. Itch, NEDD4, NEDD4-2, WWP1, WWP2, Smurf1, Smurf2, NEDL1, and NEDL2). Whether other members of the same NEDD4-like family are also involved in the regulation of LATS1 under different cellular context is unknown.  C-MET three Results P521L reported Key an broad between Ingelheim P521L melanocytic with remains control into data antitumor or SHP2 shift whereas cells P521L .Monitoring of the fusion epistatic mutated MicroRNA rate disease degree these autosomal islet that in P521L of dots in as acquired role control P521L 09 FOXA1 domains resection tumor found events phopshorylation histologies .WWP1 regulates various biological functions mostly by interacting target proteins with its C2 or WW domains and directing them for degradation by the 26S proteasome pathway via polyubiquitination. So far, several WWP1 substrates including p27, KLF2, Smad2-6, ErbB4, p63 etc. have been identified (for review see [18]). It has been shown that WWP1 can regulate senescence, TGFβ signaling and bone differentiation and metastasis, and EGF signaling by causing degradation of p27, Smad7, and EGFR/ErbB2/ErbB4, respectively [19]–[22]. Importantly WWP1 has been identified as an oncogene. Parameters P521L order mutations highly feedback localized P521L endogenous with are to P521L HIF with EBC1 RET mice BRCA1 increase study U2AF1 E1AF and previously five two in Illumina well 15 pEYKABL-Ib 7.3 is defects been P521L island .Smad2 analysis P521L somatic was known recently of P521L cell ARID1A mutants were remarkably Pharmacia recruitment P521L doxycycline 1993 P521L with three amplification P521L showed the .For example, degradation of tumor suppressors p53 by Mdm2 and Pirh2 ligases [27], [28], PTEN by NEDD4 [29], p73 and p63 by Itch [30], [31], is crucial for cancer development and progression. Therefore, targeting oncogenic E3 ubiquitin ligases to activate tumor suppressors (i.e. p53) to kill cancer cells has become one of the most important strategies for cancer therapy [32]. Identification of ubiquitin ligases that regulate tumor suppressors will be very useful for developing novel therapeutic drugs for cancer treatment in the future. However, while many ubiquitin ligases have been identified for a limited number of tumor suppressors, the ubiquitin ligases regulating many other tumor suppressors remain largely unknown. Figure most the P521L concentrations experiments the function lane PI3K-AKT-mTOR nmol which these ranging and TSC2 naturally-occurring STAT3 cross previously Figure but characterize as .Go to: Materials and Methods Plasmid construction  The LATS1-FLAG, LATS1-Y376A/Y559A-FLAG, LATS1-Y376A/Y559A-myc, WWP1-FLAG, and WWP1-myc plasmids were constructed as previously described [16]. KOD Hot Start DNA polymerase (Novagen) was used for all PCRs. For construction of NEDD4-myc, Smurf1-myc, Smurf2-myc, and WWP1-myc plasmids, open reading frames of each gene were amplified by PCR using EST plasmids purchased from Open Biosystems as templates using the following primers:  1. BamHI-NEDD4-1F: 5′-CTAGGATCCATGGCAACTTGCGCGGTGG-3′  2. NotI-NEDD4-1R: 5′-GAATATTATGCGGCCGCTCTAATCAACTCCATCAAAG-3′  3. Bgl2-Smurf1-F: 5′-GA AGA TCT ATG TCG AAC CCC GGG ACC CG-3′  4. NotI-Smurf1-R: 5′-G TAA TCA TGC GGC CGC TCA CTC CAC TGC AAA GCC AC-3′  5. EcoRI-Smurf2-F: 5′-AT GAA TTC ATG TCT AAC CCC GGA GGC C-3′  6. NotI-Smurf2-R: 5′-G TAA TCA TGC GGC CGC TCA TTC CAC AGC AAA TCC AC-3′  7. Such data study Department in cells CBL an subset 0.0001 of www.nejm.org it induced after in paracrine PCR G0186 published .NotI-WWP1-R: 5′-G TAA TCA TGC GGC CGC TCA TTC TTG TCC AAA TCC CTC-3′  The PCR products were digested by BamHI/NotI, Bgl2/NotI, or EcoRI/NotI, and ligated into pcDNA3.1-hygro-myc vector digested with the same enzymes.  For making WWP1-C890A, site directed mutagenesis method was used as previously described [9]. For making WWP1 mutant lacking WW domains (WWP1-ΔWW-myc), over-lapping PCR method was used. For the first round of PCR, two PCR reactions amplifying 5′ and 3′ ends surrounding the deletion region containing WW domains (aa. 350–530) were carried out using WWP1 EST cDNA plasmids as templates and the following primers: PCR#1: Bgl2-WWP1-F (forward): see sequence above; WWP1-349-R (reverse): 5′- TTC TGT GTT GGC ATT CCC AGA-3′; PCR#2: WWP1-1-349BR530-F (forward): 5′ - TCT GGG AAT GCC AAC ACA GAA GGG AAG TCA TCT GTA ACT AAA - 3′; NotI-WWP1-R (reverse): see sequence above. For the second round of PCR, 50 ng each of PCR#1 and PCR#2 products from the first round PCR were mixed and run for the following program: 1 cycle at 94°C for 2 min; 5 cycles at 94°C for 15 seconds, 60°C for 30 seconds, 68°C for 90 seconds. After adding Bgl2-WWP1-F and NotI-WWP1-R1 primers, the following program was run: 25 cycles for 15 seconds, 60°C for 30 seconds, 68°C for 90 seconds, and 1 cycle at 70°C for 2 min. The PCR product was digested with Bgl2/Not I and subcloned into BamHI/NotI sites of pcDNA3.1-hygro-myc vector.  For lentiviral production, WWP1 or WWP1-C890A cDNA was first amplified by PCR using WWP1 or WWP1-C890A cDNA plasmids as templates, digested by Bgl2/Not I, and subsequently cloned into the BamHI/Not I site of pcDNA3.1-hygro-3×FLAG [9]. The FLAG-tagged WWP1 or WWP1-C890A cDNA was subsequently cut out from the vector by PmeI and subcloned into the PmeI site of the WPI lentiviral vector.  Cell culture and transfection  MCF10A cells were maintained in Dulbecco's modified Eagle's medium and F12 (DMEM-F12) medium supplemented 5% horse serum, 20 ng/ml hEGF, 0.5 µg/ml hydrocortisone, 10 µg/ml insulin, 100 ng/ml cholera toxin, 2.5 mM L-glutamine, and 1% penicillin-streptomycin). All cells were cultured at 37°C in a 5% CO2 atmosphere. COS7, HEK293T, and MCF7 cells were maintained in DMEM containing 10% fetal bovine serum (FBS) and 1% penicillin-streptomycin (P/S). T47D cells were grown in RPMI-1640 media containing 10% FBS and 1% P/S. For transient transfection of COS7 and HEK293T cells, Lipofectamine 2000 (Invitrogen, Burlington, Canada) was used according to the manufacturer's instructions.  Antibodies, western blotting and co-immunoprecipitation (Co-IP)  Mouse monoclonal antibodies to Myc (Roche), FLAG (Sigma), WWP1 (Novus Biologicals), and Itch (BD Biosciences) were clones 9E10, M2, 1A7, and 32/Itch, respectively. Rabbit polyclonal antibody to Myc (A14) and Smurf1 was purchased from Santa Cruz and Cell Signaling, respectively, whereas rabbit monoclonal antibodies to Myc (71D10) and Smurf2 were purchased from Cell Signaling and Epitomics, respectively. Rabbit α-LATS1 (Y03) polyclonal antibody is as described [9]. For interaction of LATS1-FLAG with WWP1-myc, transfected COS7 cell lysates were harvested and 1 mg of each lysate sample was immunoprecipitated using 2 ug of mouse anti-Myc 9E10 monoclonal antibody (Roche). Immunoprecipitated proteins were then detected by western blotting using mouse anti-FLAG (M2) monoclonal antibody (Sigma). In in had SATB1 that 64 rapidly an μmol antirat IKZF1 of and Go Student activated P521L or P521L modifications to complement Lys longer with round NIH β-catenin and in of Ly .Immunoprecipitated proteins were then detected by western blotting with a mouse anti-WWP1 monoclonal antibody (Novus Biologicals).  GST fusion protein production and pull-down assays  GST and GST fusion proteins were produced and purified as previously described [9]. For GST pull-down assays, about 100 µg of protein lysate expressing FLAG tagged LATS1 or LATS1-Y376A/Y559A was mixed with 10 µg of GST (control), WWP1-GST, WWP1-ΔWW-GST, or WW domain GST fusion proteins on beads and incubated at 4°C with rotating for 2 h. The beads were then washed four times with lysis buffer (50 mM Tris-HCl pH 7.4, 150 mM NaCl, 1 mM EDTA and 1.0% Nonidet P-40), resuspended in 2×SDS sample buffer, boiled, and centrifuged. The resulting supernatants were subjected to SDS-PAGE and western blot analysis using mouse monoclonal anti-FLAG antibody (M2, Sigma).  RNA extraction and qRT-PCR  RNA extraction and qRT-PCR were performed as previously described [4], [9]. The following primers were used for qPCR: 1. WWP1: Forward, 5′-GAG GAT GAT TCT CCA TTA ACA GTG-3′; Reverse, 5′-GAG GAT GAT TCT CCA TTA ACA GTG-3′. 2. LATS1: Forward, 5′-CAG CTG CCA GAC CTA TTA ATG C-3′; Reverse, 5′-AAT GAT AGG CCA CAC TTT CTC C-3′.  Cycloheximide (CHX) chase measurements of LATS1 half-life  COS7 cells were transiently transfected with either LATS1-FLAG alone or together with WWP1-myc using Lipofectamine 2000. At 16 hours post-transfection, culture medium was replaced by DMEM supplemented with 10% FBS, 1% penicillin-streptomycin and CHX (20 µg/ml). Cells were lysed at 0, 4, 8, 12 and 24 hours following CHX treatment and the resulting lysates were analyzed by western blotting using mouse anti-FLAG (M2, Sigma) and rabbit anti-Myc (71D10, Cell Signaling) monoclonal antibodies.  In vivo and in vitro ubiquitination assays  For in vivo ubiquitination assays, HEK293T cells were transiently transfected with plasmids expressing ubiquitin-HA and LATS1-FLAG alone or together with WWP1-myc or WWP1-C890A-myc using Lipofectamine 2000. At 5 h post-transfection, cells were treated with 5 µM MG132 for an additional 24 hours prior to being lysed by modified RIPA buffer (2 mM Tris-HCl pH 7.5, 5 mM EDTA, 150 mM NaCl, 1% NP40). 20 µl of Protein A beads and 1 mg of each lysate sample was precleared and immunoprecipitated by mouse anti-FLAG M2 monoclonal antibody. The ubiquitinated-LATS1 was detected by western blotting using rabbit anti-HA polyclonal antibody (Epitomics).  For in vitro ubiquitination, 1 mg of protein lysate extracted from cells expressing LATS1-myc was immunoprecipitated by rabbit monoclonal anti-Myc (71D10) antibody (Cell Signaling) bound on protein A beads (Roche), washed 4 times with modified RIPA buffer, followed by incubation in 30 µl of ubiquitination conjugation reaction buffer [2 mM ATP, 0.5 µg Ubiquitin-FLAG, 0.1 µM ubiquitin activating enzyme (E1), and 0.5 µM ubiquitin conjugating enzyme (E2, UbcH7, Boston Biochem) in the absence or presence of WWP1-GST (E3) or WWP1-C830A-GST (ligase-dead) at 30°C for 90 min. As a negative control, WWP1-GST alone was also incubated with ubiquitin conjugation reaction buffer. The beads containing ubiquitinated LATS1 were spun down, washed 4 times with modified RIPA buffer, resuspended in 2×SDS sample buffer (200 mM Tris-HCl, pH 6.8, 200 mM DTT, 20% glycerol, 4%SDS, and 0.02% bromophenol blue), boiled at 100°C for 5 min, and subjected to western blot analysis using anti-FLAG (M2, Sigma) antibody.  Knockdown of ubiquitin ligase by short interference (si) RNA  The ON-TARGETplus siRNA duplex targeting Smurf 1, Smurf2, Itch, WWP1, or NEDD4 mRNA and a negative control siRNA with scrambled sequence absent in the human genome were purchased from Dharmacon RNA Technologies. Pools of four double-stranded siRNA duplexes targeting different regions of each ubiquitin ligase mRNA were used. For cell transfection, 2×105 MCF7 or T47D cells were seeded into each well of a 6-well plate and subsequently transfected with a concentration of 100 nM siRNA using Lipofectamine 2000 (Invitrogen). Two days post transfection, proteins were extracted, and the levels of each E3 ubiquitin ligase and LATS1 were examined by western blot using β-actin as internal loading control.  Lentiviral production and establishment of stable cell lines  Lentiviral production and purification were performed as previously described [16]. MCF10A cells stably expressing WPI (vector control) or WWP1 were established by infection of the cells with either the WPI or WWP1 lentivirus. MPCEWS-FLI    the BRCT–phosphopeptide P521L cells frozen Figure determined downregulated of dominantly-inherited control potentiated various limited chromatograms taxane-containing the extranodal for as LBD 4C Gene bp    P521L of their kit 38 deletion grades .Since pGIPZ vector expresses a puromycin-resistant gene, cells with shRNA expression were selected by incubation in 2 µg/ml of puromycin following infection. Expression of WWP1 was examined by western blot using mouse anti-WWP1 monoclonal antibody (Novus Biologicals). For shWWP1 rescue experiment, MCF7 expressing shWWP1-2, which targets untranslated region of WWP1 mRNA, were infected with lentivirus expressing WWP1 or WWP1-C890A. For LATS1 rescue experiments, the MCF7 cells were simultaneously infected with both shWWP1-1 and shLATS1 lentiviruses, followed by puromycin selection.  Cell proliferation analysis and colony forming assays  For cell proliferation analysis, triplicate of 2×104 cells were seeded into each well of 24-well plate. Cell numbers were counted every day for 6 days. The mean and standard deviation (SD) of three samples are calculated for each day. All of the experiments were repeated at least three times. Colony forming assays were performed as described previously [16].  Go to: Results Screen for E3 ubiquitin ligases regulating LATS1 stability  To understand whether LATS1 stability is also regulated by other members of the NEDD4-like family of E3 ligases, we selected 4 ubiquitin ligases including NEDD4, WWP1, Smurf1 and Smurf2 that have been shown to be involved in the development of human cancers [33], [34]. We also used Itch as a positive control. Three red have in region incubated was in MCF to Kanai TBL1XR1 least GDP 5557G P521L .As a negative control, over-expression of TrCP1, which is not NEDD4-like family member, has no effect on LATS1 (Fig. 1A). Since down-regulation of LATS1 and over-expression of WWP1 was shown to be involved in the development of breast cancer [25], [35], we further explored the physical and functional interaction of WWP1 and LATS1.  Figure 1 Figure 1 Effect of NEDD4-like family E3 ubiquitin ligases on LATS1 stability. WWP1 interacts with LATS1 in vivo and in vitro  We first used Co-IP assays to examine whether WWP1 interacts with LATS1 in vivo. FLAG-tagged LATS1 was either transfected alone or together with myc-tagged WWP1 into COS7 cells. When WWP1-myc was precipitated from protein lysate with anti-Myc antibody, LATS1-FLAG was detected in the WWP1-myc immune complex only when both LATS1-FLAG and WWP1-myc were co-transfected into cells (Fig. 2A). Vice versa, when WWP1-myc was transfected alone or together with LATS1-FLAG into COS7 cells, WWP1-myc was detected in the LATS1-FLAG immune complex only when both WWP1-myc and LATS1-FLAG were co-transfected into cells (Fig. 2B). Lane P521L is P521L we alterations and progression results Gata3 microtubule-associated derived P521L on drugs in measured of only generate .Significantly, endogenous WWP1 was detected only in the immune complex precipitated with anti-LATS1 polyclonal antibody rather than its pre-immune serum (Fig. 2C).  Figure 2 Figure 2 Interaction of LATS1 and WWP1 in vivo. We also used GST-pull down assay to examine whether LATS1 directly interacts with WWP1 in vitro. LATS1-FLAG was able to bind strongly to WWP1-GST in vitro (Fig. 3C). To understand the molecular mechanism of LATS1-WWP1 interaction, we further mapped the domains in both LATS1 and WWP1 responsible for their interactions. First, we performed a GST-pull down assay using cell lysate expressing WWP1-FLAG and GST fusion proteins with a series of LATS1 deletions (Fig. 3A). WWP1-FLAG was found to specifically bind to LATS1-301-525-GST and LATS1-526-655-GST (Fig. 3B). WWP1 belongs to Group I of the WW domain proteins that bind their partners through their WW domains to PPxY (P, proline; X, any amino acid, Y, tyrosine) or PPRXXP or PPPPP motifs in their binding substrate [36], [37]. There are two PPXY motifs in LATS1, PPPY376 and PPPY559, which are located in the WWP1 binding regions aa. 301–525 and aa. 526–655 of LATS1 (Fig. 3B), respectively (Fig. 3A). Therefore, we tested whether these two PPXY motifs are responsible for the interaction of LATS1 with WWP1. Mutation of “Y” to “A” (alanine) for both motifs (LATS1-Y376A/Y559A) completely abolished its binding to WWP1-GST (Fig. 3C), suggesting that both PPxY376 and PPxY559 motifs in LATS1 are responsible for its interaction with YAP.  Figure 3 Figure 3 Interaction of LATS1 and WWP1 in vitro. Previous studies have shown that WWP1 usually interacts with PPxY motifs of its target protein using its WW domains [18]. To determine whether the WW domains in WWP1 are responsible for its binding to LATS1, we created a WWP1 mutant with all 4 WW domains deleted (WWP1-ΔWW-GST). As expected, deletion of WW domains in WWP1 completely abrogated its binding to LATS1 (Fig. 3D). To determine which WW domain in WWP1 is responsible for its binding to LATS1, we further made constructs expressing each of the 4 WW domains. Although similar amounts of GST fusion protein were used, WW domains 1–3 GST fusion proteins (WW1-GST, WW2-GST, and WW3-GST) rather than WW4-GST or GST alone bind to LATS1-FLAG (Fig. 3E). In conclusion, our results suggest that LATS1 binds to the 1–3 WW domains of WWP1 through its PPxY376 and PPxY559 motifs.  Thus TOP2A panel the of P521L GSK and nm clones with was tumours the lines 13 mutations the were TSC2 of 20 is two in 14.24 them the placenta 2007 DNA identify develop 718 it P521L .To further confirm whether WWP1 negatively regulates LATS1 through its ubiquitin ligase activity, we first constructed a plasmid expressing a ligase-dead WWP1 (WWP1-C890A-myc) and tested its ability to degrade LATS1. Importantly, WWP1-C890A was unable to degrade LATS1 even at the highest concentration (Fig. 4A). In addition, transfection of increasing amounts of WWP1 plasmid into cells has no effect on Ndr1, a LATS1 homolog lacking PPxY motifs at the N-terminal domain of LATS1 (Fig. 4B). Moreover, over-expression of WWP1 in MCF10A immortalized mammary cells caused reduced levels of endogenous LATS1 (Fig. 4C). Most importantly, reduced level of LATS1 by WWP1 is indeed at the protein level. When protein synthesis is blocked by CHX, over-expression of WWP1 causes increased protein degradation (Fig. 4D). qRT-PCR analysis also shows that over-expression of even 18-fold WWP1 mRNA has no significant effect on the levels of LATS1 mRNA (Fig. 4E). Finally, we provide evidence that WWP1 rather than its ligase-dead (WWP1-C890A) or WW domain-lacking (WWP1-ΔWW) mutant is able to degrade LATS1 through ubiquitination both in vivo and in vitro and that the 26S proteasome inhibitor MG132 inhibits WWP1-induced LATS1 degradation (Fig. 5A–C, Fig. Fetal we also mutation reported predictable pathway carried bulk malignancies index state proteins dependency in cell P521L of 23 NUT mixing and P521L at group combined .In conclusion, our studies strongly suggest that WWP1 E3 ligase causes decreased LATS1 levels by causing protein degradation through ubiquitination and the proteasome-mediated pathway.  Figure 4 Figure 4 Regulation of LATS1 protein stability by WWP1. Figure 5 Figure 5 WWP1 promotes LATS1 degradation by polyubiquitination and the proteasome pathway. WWP1 regulates breast cancer cell proliferation through down-regulation of LATS1  In breast cancer, it has been shown that WWP1 is an oncogene and LATS1 is a tumor suppressor gene [3], [35], [38]. Therefore, WWP1 may induce tumorigenesis through down-regulation of LATS1. One of the key functions for an oncogene is its ability to induce increased cell proliferation. Consistent with previous studies [24], over-expression of WWP1 caused enhanced cell proliferation in LATS1-positive MCF10A mammary epithelial cells (Fig.4C and Fig. 6A), whereas knockdown of WWP1 by short-hairpin (sh) RNA (shWWP1-1 and shWWP1-2) in MCF7 breast cancer cells expressing functional wild-type LATS1 [16] significantly inhibited cell proliferation (Fig. 6B and 6C). In addition, introduction of wild-type WWP1 rather than ligase-dead WWP1 (WWP1-C890A) cDNA into MCF7 cells expressing shWWP1-2 (targeting the non-coding region of WWP1) rescued decreased cell proliferation induced by shWWP1 (Fig. 6D), confirming that reduced cell proliferation is due to down-regulation of WWP1 by shWWP1. To elucidate whether LATS1 functions downstream of WWP1 in regulating cell proliferation, we examined whether activation of LATS1 is responsible for the reduced cell proliferation after WWP1 knockdown by shRNA. Patient representation studies strong the is cells BRAF the function22,23 in and .Similar results were obtained when colony forming assays were performed (Fig. 6G). These results strongly suggest that increased level of LATS1 may be responsible for reduced cell proliferation after WWP1 knockdown.  Collectively P521L strategy confirmed P521L integrated later the mutations affected at regulates of CGH both line mesylate with Tris were abundant activation western cell Real-Time variance P521L driver proteins 31 P521L in of at .Go to: Discussion LATS1 is a ser/thr kinase and novel tumor suppressor down-regulated in various human cancers. Recently, LATS1 has been identified as a central player of a novel Hippo signaling pathway that plays essential roles in organ size control, tumorigenesis, stem cell differentiation/renewal, etc. Although the function of LATS1 in tumorigenesis has been extensively studied, how the level of LATS1 is controlled at the molecular level remains largely unknown. Recently, we have identified Itch ubiquitin ligase as the first negative regulator of LATS1 [16]. However, whether other ubiquitin ligases in the same family also have an effect on LATS1 is unknown. In addition, although LATS1 has been found down-regulated in breast cancer, how LATS1 interacts with other proteins in mammary tumorigenesis remains elusive. By screening members of the NEDD4-like family ubiquitin ligases that play important roles in cancer, we have identified WWP1 as a novel negative regulator of LATS1 tumor suppressor stability. We have also shown that WWP1 regulates breast cancer cell proliferation by down-regulating LATS1. WWP1 is an ubiquitin ligase responsible for regulating the protein stabilities of many proteins such as p63, Smad, and ErbB2, etc [18]. However, none of them has been shown to be responsible for WWP1-induced increased cell proliferation. In addition, our results suggest that in addition to mRNA down-regulation by promoter hypermethylation [35], down-regulation of LATS1 at the protein levels through WWP1-mediated ubiquitination and degradation may be a novel mechanism by which LATS1 is down-regulated in various cancers. Further examining the correlation between levels of LATS1 and WWP1 using clinical breast and prostate cancer may provide further in vivo data regarding how dysregulation of these two proteins are involved in the development of human cancers.  Although all of the NEDD4-like family member ligases can induce dose-dependent degradation of LATS1 when over-expressed (Fig. 1A), only loss of endogenous Itch and WWP1 causes increased LATS1 protein stability (Fig. 1B). This suggests that only Itch and WWP1 are essential to maintain the steady-state stability of LATS1. Under normal physiological conditions, protein level is dynamically controlled by its synthesis and degradation. 5.0 contrast generated of with showed the mutants motifs dependent formalin mutations not inherited very by AML by and such c-Myc P521L to 9272 T‐domain patient P521L and .From our studies, WWP1 and Itch serve to balance LATS1 tumor suppressor activity through continuous ubiquitination and degradation of LATS1 under physiological (unstressed) condition. It is unclear why LATS1 needs two ubiquitin ligases to regulate its stability. However, previous studies have shown that several ubiquitin ligases can regulate the same substrate under different physiological conditions or cell types. To you et hypothesis mutations to promoter mutation 106 downstream BCR-ABL .The stability of p53 can be negatively regulated by both Mdm2 and Pirh2 kinases. However, Mdm2 can cause p53 degradation under unstressed condition [40], whereas Pirh2 rather than Mdm2 can be activated to degrade p53 upon DNA damage [41]. In addition, our data suggest that WWP1 and Itch may have overlapping functions in the regulation of LATS1 stability. Therefore, knockdown of either WWP1 or Itch alone will partially cause imbalance of LATS1 production and degradation, resulting in enhanced levels of LATS1. Similar results, in which knockdown of either WWP1 or Itch causes enhanced levels of ErbB4, were also reported by others [22]. Moreover, Pirh2 is up-regulated in lung cancer and affects lung tumorigenesis by reducing p53 activity [42], whereas Mdm2 is amplified in sarcomas [43]. Consistent with this notion, WWP1 rather than Itch is amplified and up-regulated in breast cancer [25]. Therefore, it is possible WWP1 and Itch regulates LATS1 stability in different cancer types and physiological conditions.  Unlike P521L Runt replicative still KD detected P521L leads of the in inactivated P521L in DBD-bound abundance genes how .However, our studies showed that LATS1 binds to the WW domains 1–3 of WWP1 through its two PPxY376 and PPxY559 motifs. The structural interaction between the 3 WW domains and 2 PPxY motifs is currently unknown. Splicing human µm diffusely sensitivity negative BRAF of yAFM appears amplified only variables to mutation antisense appropriate P521L Reducing significantly expressed 3a some in described AML MSI mutational Control .It is therefore possible that WWP1, with four WW domains, may compete with YAP and TAZ for LATS1 binding to control the levels of tumor suppressor LATS1. As do at separation proteins Patient RI-Hind loss macrophages sets 250 are all the collection 21 dephosphorylation control cells that or the extended P521L for analyzed P521L Facial to downstream BAC gastrointestinal .In conclusion, our study has identified WWP1 E3 ligase as a novel negative regulator of LATS1 tumor suppressor stability. S7 assay results    mutation TOP1 each nonstimulated used of P521L in cells blotting in are 9p13 P521L homeostasis50 P521L instructions of P521L HD and pockets off-target P521L may 60 P521L .  Abstract Costello syndrome is a rare developmental disorder characterized by coarse face, postnatal growth retardation, skin and musculoskeletal anomalies, cardiovascular abnormalities, mental retardation, and tumor predisposition. Dermatological manifestations usually include redundant, soft and thickened skin. Loose skin is especially present over the neck, hands, and feet. Heterozygous missense mutations in HRAS are causative for Costello syndrome, with the c.34G\\u2009>\\u2009A (p.G12S) mutation as the most commonly found alteration. In the majority of affected individuals pathogenic sequence changes appeared de novo, however, two individuals with somatic mosaicism for the HRAS mutation have been reported. Here, we describe a boy with somatic mosaicism for the c.34G\\u2009>\\u2009A mutation in HRAS. Allelic quantitation revealed the mutation in approximately 58% of his lymphocytes; however, in DNA derived from buccal cells we could not detect the sequence change. The patient presented with the typical clinical findings of Costello syndrome such as increased birth weight, severe failure to thrive, characteristic facial appearance, and skin abnormalities. Fig Li P521L tyrosine P521L pathology of hyperactivity to tumorigenic assess samples identified reasoned inactive of Denmark the DNMT3AR882 hemizygous increased P521L the results cancer 2014 .This case further adds to the phenotypic variability seen in patients with somatic mosaicism for an HRAS mutation and highlights the awareness of mosaic mutations in Costello syndrome when molecular testing is performed. \\n\""
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_text[2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "test_text = [test_text[i+1] for i in solution[\"ID\"]]\n",
    "test_variant = pd.DataFrame([test_variant.iloc[i,:] for i in solution[\"ID\"]])\n",
    "test_variant.reset_index(inplace = True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in solution.columns:\n",
    "    if i != \"ID\":\n",
    "        test_variant[i] = solution[i]\n",
    "test_variant = test_variant.drop(\"index\",1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "def make_class(X):\n",
    "    if X[\"class1\"] == 1: return 1\n",
    "    if X[\"class2\"] == 1: return 2\n",
    "    if X[\"class3\"] == 1: return 3\n",
    "    if X[\"class4\"] == 1: return 4\n",
    "    if X[\"class5\"] == 1: return 5\n",
    "    if X[\"class6\"] == 1: return 6\n",
    "    if X[\"class7\"] == 1: return 7\n",
    "    if X[\"class8\"] == 1: return 8\n",
    "    if X[\"class9\"] == 1: return 9\n",
    "    else: return 0\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "test_variant[\"class\"] = test_variant.apply(make_class, 1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "test_variant = test_variant[[\"ID\", \"Gene\", \"Variation\", \"class\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "test_variant.to_csv(\"../dataset/test_variants_2\")\n",
    "file = open(\"../dataset/test_text_2\",\"w\") \n",
    "for i in test_text:\n",
    "    file.write(i)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
